### PR TITLE
github-event-types.md : add missed event object common properties into list

### PR DESCRIPTION
**Pull request by NickVeld**
### Why:

Closes  https://github.com/github/docs/issues/18166

### What's being changed:

`public`, `created_at`, `org.id`, `org.login`, `org.gravatar_id`, `org.url`, `org.avatar_url` are added using the same ordering as used on https://api.github.com/events

Added after the following line (for content view stability the last commit for now is chosen for the link): https://github.com/github/docs/blob/6189de4d25c0380247fa73ee5c4767c1129e9818/content/developers/webhooks-and-events/events/github-event-types.md?plain=1#L40

The code snippet of the addition:
```
| `public` | The property shows whether the event can be shown to all API caller or not. |
| `created_at` | The date and time when the event was triggered. It is formatted according to ISO 8601. |
| `org` | The organization that was chosen by the actor to perform action that triggers the event. _The property appears in the event object only if it is applicable._ |
| `org.id` | The unique identifier for the organization. |
| `org.login` | The name of the organization. |
| `org.gravatar_id` | The unique identifier of the Gravatar profile for the organization. |
| `org.url` | The REST API URL used to retrieve the organization object, which includes additional organization information. |
| `org.avatar_url` | The URL of the organization's profile image. |
```

The table from "Automatically generated comment" with the comment:
| **Source** | **Preview** | **Production** | **What Changed** |
|-------------|--------------|------------------|----------------------|
| [`developers/webhooks-and-events/events/github-event-types.md`](https://github.com/github/docs/blob/b8a70f47f11cc291e3c8e20f47e9ba1a883288e1/content%2Fdevelopers%2Fwebhooks-and-events%2Fevents%2Fgithub-event-types.md) | [fpt](https://docs-18167-fd8d48.preview.ghdocs.com/developers/webhooks-and-events/events/github-event-types)<br /> [ghec](https://docs-18167-fd8d48.preview.ghdocs.com/enterprise-cloud@latest/developers/webhooks-and-events/events/github-event-types)<br /> ghes@ [3.5](https://docs-18167-fd8d48.preview.ghdocs.com/enterprise-server@3.5/developers/webhooks-and-events/events/github-event-types) [3.4](https://docs-18167-fd8d48.preview.ghdocs.com/enterprise-server@3.4/developers/webhooks-and-events/events/github-event-types) [3.3](https://docs-18167-fd8d48.preview.ghdocs.com/enterprise-server@3.3/developers/webhooks-and-events/events/github-event-types) [3.2](https://docs-18167-fd8d48.preview.ghdocs.com/enterprise-server@3.2/developers/webhooks-and-events/events/github-event-types) [3.1](https://docs-18167-fd8d48.preview.ghdocs.com/enterprise-server@3.1/developers/webhooks-and-events/events/github-event-types)<br /> [ghae](https://docs-18167-fd8d48.preview.ghdocs.com/github-ae@latest/developers/webhooks-and-events/events/github-event-types) | [fpt](https://docs.github.com/developers/webhooks-and-events/events/github-event-types)<br /> [ghec](https://docs.github.com/enterprise-cloud@latest/developers/webhooks-and-events/events/github-event-types)<br /> ghes@ [3.5](https://docs.github.com/enterprise-server@3.5/developers/webhooks-and-events/events/github-event-types) [3.4](https://docs.github.com/enterprise-server@3.4/developers/webhooks-and-events/events/github-event-types) [3.3](https://docs.github.com/enterprise-server@3.3/developers/webhooks-and-events/events/github-event-types) [3.2](https://docs.github.com/enterprise-server@3.2/developers/webhooks-and-events/events/github-event-types) [3.1](https://docs.github.com/enterprise-server@3.1/developers/webhooks-and-events/events/github-event-types)<br /> [ghae](https://docs.github.com/github-ae@latest/developers/webhooks-and-events/events/github-event-types) | All preview links show the addition of the properties as expected |

### Check off the following:

- [x] I have reviewed my changes in staging (look for "Automatically generated comment" and click **Modified** to view your latest changes).
- [x] For content changes, I have completed the [self-review checklist](https://github.com/github/docs/blob/main/contributing/self-review.md#self-review).

### Writer impact (This section is for GitHub staff members only):

- [ ] This pull request impacts the contribution experience
  - [ ] I have added the 'writer impact' label
  - [ ] I have added a description and/or a video demo of the changes below (e.g. a "before and after video")

### CODE REVIEW

**Comment by janiceilene** [comment]
@NickVeld Thanks so much for opening a PR! I'll get this triaged for review ⚡

File: content/developers/webhooks-and-events/events/github-event-types.md [hunk 1]
```diff
@@ -38,6 +38,14 @@ The event objects returned from the Events API endpoints have the same structure
...
 | `repo.url` | The REST API URL used to retrieve the repository object, which includes additional repository information. |
 | `payload` | The event payload object is unique to the event type. See the event type below for the event API `payload` object. |
+| `public` | The property shows whether the event can be shown to all API callers or only authorized ones. |
+| `created_at` | The date and time when the event was triggered. It is formatted according to ISO 8601. |
+| `org` | The organization that was chosen by the actor to perform action that triggers the event.<br />_The property appears in the event object only if it is applicable._ |
+| `org.id` | The unique identifier for the organization. |
+| `org.login` | The name of the organization. |
+| `org.gravatar_id` | The unique identifier of the Gravatar profile for the organization. |
+| `org.url` | The REST API URL used to retrieve the organization object, which includes additional organization information. |
+| `org.avatar_url` | The URL of the organization's profile image. |
```
Comment (skedwards88) [inline content/developers/webhooks-and-events/events/github-event-types.md:48]:
```suggestion
| `public` | Whether the event is visible to all users. |
| `created_at` | The date and time when the event was triggered. It is formatted according to ISO 8601. |
| `org` | The organization that was chosen by the actor to perform action that triggers the event.<br />_The property appears in the event object only if it is applicable._ |
| `org.id` | The unique identifier for the organization. |
| `org.login` | The name of the organization. |
| `org.gravatar_id` | The unique identifier of the Gravatar profile for the organization. |
| `org.url` | The REST API URL used to retrieve the organization object, which includes additional organization information. |
| `org.avatar_url` | The URL of the organization's profile image. |
```

**Review by skedwards88 — COMMENTED** [review]
Thank you for this addition! I made a minor suggestion that I will incorporate, then we'll get this merged down for you.

**Review by skedwards88 — APPROVED** [review]

**Comment by NickVeld** [comment]
Thanks to all who participated in the PR 👍

**Pull request merged**